direct/tpm2/read_public_test.go

```diff
@@ -13,14 +13,20 @@ import (
 	"github.com/google/go-tpm/direct/transport/simulator"
 )
 
+// TestReadPublicKey compares the createPrimary PublicArea when instantiated with
+// the PublicArea read from executing readPublic.
 func TestReadPublicKey(t *testing.T) {
-
+	// Open simulated TPM for testing.
 	thetpm, err := simulator.OpenSimulator()
 	if err != nil {
 		t.Fatalf("could not connect to TPM simulator: %v", err)
 	}
+	
+	// Defer the close of the simulated TPM to after use.
 	defer thetpm.Close()
 
+	// Fill in the CreatePrimary struct.
+	// See definition in Part 3, Commands, section 24.1.
 	createPrimary := CreatePrimary{
```

Comment:
did we ever create test helper templates or Publics? If so, add a pointer to them.

Reply:
Added. Resolved.